Update oracle-connection-manager.md (#9021)
MicrosoftDocs/sql-docs: docs/integration-services/data-flow/oracle-connection-manager.md

The **ConnectionManagerType** property for the Oracle Connection Manager is set to **ORACLE**.

> In SSIS execution logs this connector will be refered to as "Oracle Connection Manager".

Reviewer: ```suggestion
In SSIS execution logs, this connector is referred to as "Oracle Connection Manager."
```
